### DISCUSSION TITLE
Copy & Paste

### DISCUSSION THREAD

**Discussion by PeterKnotek** [immense/Remotely#943] [General]
Hi all 
Copy & Paste not work 
Have latest version remotely 
Is normal or i have issue ??

Thaks all

**Comment by PeterKnotek** [comment]
nobody nothing ??

**Reply by TheFifthBrother** [reply]
What are you trying to Copy?
if it just text you paste it from the top drop down bar
![image](https://github.com/user-attachments/assets/0e1707a5-f322-432f-b799-4d55b3f9c9cd)

Make sure you curser is where you want the text to be then click type Clipboard.

As for anything such as files or folders you need to use the file transfer options which is found on the right hand side.
![image](https://github.com/user-attachments/assets/b49d0a61-efe9-4e36-beb6-474b8fdf4798)


Hope this helps as I am rather new. But I haven't had any problems with those features.

**Reply by PeterKnotek** [reply]
HI

Clipboard API only works in a secure context (i.e. HTTPS or localhost). ....
